projects/ui/src/lib/components/po-tag/po-tag-base.component.spec.ts

```diff
@@ -7,25 +7,60 @@ import { PoTagType } from './enums/po-tag-type.enum';
 
 describe('PoTagBaseComponent:', () => {
   const component = new PoTagBaseComponent();
+  const poTagColors = [
+    'color-01',
+    'color-02',
+    'color-03',
+    'color-04',
+    'color-05',
+    'color-06',
+    'color-07',
+    'color-08',
+    'color-09',
+    'color-10',
+    'color-11',
+    'color-12',
+  ];
 
   it('should be created', () => {
     expect(component instanceof PoTagBaseComponent).toBeTruthy();
   });
 
   describe('Properties:', () => {
 
-    it('icon: should update with true value.', () => {
+    it('color: should update to true value.', () => {
+      const colorsValidTrueValues = poTagColors;
+
+      expectPropertiesValues(component, 'color', colorsValidTrueValues, colorsValidTrueValues);
+    });
+
+    it('color: shouldn´t update to false value.', () => {
+      const colorsInalidTrueValues = [undefined, null, 2, 'string', 0, NaN];
+
+      expectPropertiesValues(component, 'color', colorsInalidTrueValues, undefined);
+    });
+
+    it('icon: should update to true value with `type`.', () => {
+      component.type = PoTagType.Info;
       const booleanValidTrueValues = [true, 'true', 1, ''];
 
       expectPropertiesValues(component, 'icon', booleanValidTrueValues, true);
     });
 
-    it('icon: should update with false value.', () => {
+    it('icon: should update to false value with `type`.', () => {
+      component.type = PoTagType.Info;
       const booleanInvalidValues = [undefined, null, 2, 'string', 0, NaN];
 
       expectPropertiesValues(component, 'icon', booleanInvalidValues, false);
     });
 
+    it('icon: should update to true value without `type`.', () => {
```

Comment:
Verificar a grafia da descrição do teste:

Sugestão:
icon: should update to true if `type` is undefined